modules/Material/Action.qml

```diff
@@ -69,6 +69,11 @@ QtObject {
     property bool visible: true
 
     /*!
+       Set to \c true to rotate the icon on mouse hover.
```

Comment:
added the number of degrees i don't think the direction is very important, we might need to add more customization to this for icons that are not simetric we should rotate 180 deg or 360